frontend/src/Map/UseGeoLocation.js

```diff
@@ -0,0 +1,42 @@
+import { useState, useEffect } from 'react';
+
+const useGeoLocation = () => {
```

Comment:
Mondtam, hogy van erre kész hook: https://github.com/streamich/react-use/blob/master/docs/useGeolocation.md
Teljesen felesleges volt leimplementálni, valamint van pár dolog, amire nem is gondoltatok...

Reply:
Sajnos későn vettem észre, a segítséget. Addigra már 6 órát beleöltem hogy rájöjjek hogy csak az App.jsben működik a kódom.